Assets/Scenes/Scripts/PlayerControl.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+[System.Serializable]
+public class PlayerControl : MonoBehaviour 
+{
+	[SerializeField]
+	private GameObject shot;
+	[SerializeField]
+	private Transform shotSpot;
+	[SerializeField]
+	private float fireRate;
+	[SerializeField]
+	private GameObject playerEx;
+	[SerializeField]
+	private HealthBar playerHealthBar;
+	[SerializeField]
+	private int Health;
+
+	public static bool Damaged;
+	private float nextfire;
+	private const float speed = 5;
+	private const float tilt = 7;
+	private Rigidbody rigidBody;
+	private AudioSource audioSource;
+
+	private void Awake()
+	{
+		Assert.IsNotNull (shotSpot);
+		Assert.IsNotNull (shot);
+		Assert.IsNotNull (playerEx);
+		Assert.IsNotNull (playerHealthBar);
+
+		rigidBody = GetComponent<Rigidbody> ();
+		audioSource = GetComponent<AudioSource> ();
+
+		Assert.IsNotNull (rigidBody);
+		Assert.IsNotNull (audioSource);
+	}
+
+	private void Start () 
+	{
+		Damaged = false;
+		nextfire = 0;
+		playerHealthBar.SetHealthValue (Health);
+	}
+
+	private void Damage()
+	{
+		--Health;
```

Comment:
This can be a Getter/Setter. Look at this documentation: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties Then you can just have Damage just `--Health;`